docs/ui/components/Markdown/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { css, CSSObject } from '@emotion/react';
+import { typography, theme } from '@expo/styleguide';
+import React, { ComponentType, PropsWithChildren } from 'react';
+
+import { A, H1, H2, H4, H5, CODE, P, BOLD, UL, OL, LI } from '~/ui/components/Text';
+
+type Config = ConfigStyles & {
+  Component: ComponentType<ComponentProps> | string;
+};
+
+type ConfigStyles = {
+  css?: CSSObject;
+  style?: React.CSSProperties;
+};
+
+type ComponentProps = PropsWithChildren<{
+  className?: string;
+  style?: React.CSSProperties;
+}>;
+
+const headerMarginBottom = '0.5ch';
+const paragraphMarginBottom = '1ch';
+const headerPaddingTop = '1.5ch';
+
+const markdownStyles: Record<string, Config | null> = {
+  // When using inline markdown, we need to remove the document layout wrapper.
+  // Always set this to `null` to overwrite the global MDX provider.
+  wrapper: null,
+  h1: {
```

Comment:
These headers feel too large to me. Here's a before (left) and after (right):

<img width="2240" alt="Screen Shot 2022-01-30 at 12 35 03 AM" src="https://user-images.githubusercontent.com/6455018/151688042-e8886289-790c-4670-968e-6f6ca726a43d.png">

One of the ways i made the design's content feel cleaner/better was by decreasing the sizes of the headers. For instance, a documentation title is an H3 (instead of an H1).

While we _can_ write components like `<H3 tag="h1">`, that doesn't seem right to me.

The typography I designed for the website is also used for marketing pages, which need to be bigger, whereas docs/long-form content should be smaller/more consistent. So! before we merge typography stuff, let me spend a little time working on setting some more typography. I think we need two sets, one for marketing and one for non-marketing pages.

Can we hold off on this PR so that I can set those styles and get them merged into /styleguide?

Reply:
Yes! It's stacked on the typography PR, so we need to merge that before we can merge this. We could also try and wing it in the typography generation, where we still have `<H2>` (without `tag=h3`), but it has the H3 styling instead of H2?

I think I actually did that here too: https://github.com/byCedric/expo/blob/docs-redesign/docs/ui/components/Text/index.tsx#L54

Reply:
Solved with the new header sizes in styleguide